app/assets/stylesheets/main.scss

```diff
@@ -97,7 +97,7 @@
 }
 
 #api_key_box {
-  @include border-radius(5px);
+  border-radius: $radius;
```

Comment:
Properties should be ordered border, border-radius, margin-top, overflow, padding